gnucash/gnome/gnc-plugin-page-invoice.c

```diff
@@ -218,20 +218,15 @@ static GtkActionEntry gnc_plugin_page_invoice_actions [] =
         G_CALLBACK (gnc_plugin_page_invoice_cmd_new_invoice)
     },
     {
-        "BusinessAssociationAction", NULL, "_Update Association for Invoice", NULL,
-        "Update Association for current Invoice",
+        "BusinessAssociationAction", NULL, "_Manage Associated Document...", NULL,
```

Comment:
We want to switch to "…" ([altgr]+[.] on my keyboard)

Reply:
OK, but I think that should be a separate commit that fixes all of them. And since it will blow up every translation it should go on master.